modules/jolt_physics/SCsub

```diff
@@ -6,6 +6,10 @@ Import("env_modules")
 
 env_jolt = env_modules.Clone()
 
+# Platform specific flags.
+if env["platform"] == "web" and env["simd"]:
+    env_jolt.Append(CCFLAGS=["-msse4.2"])
```

Comment:
Add a comment to explain why this is done (it's not immediately obvious why it's SSE 4.2 specifically). I assume this is because SSE 4.2 is the highest version supported by WebAssembly, as newer instruction sets are not supported on all CPUs.

By the way, how does ARM support work if we're specifying SSE 4.2?

```suggestion
if env["platform"] == "web" and env["simd"]:
	# Enable SSE 4.2 so that WebAssembly SIMD can benefit from it.
	# SSE 4.2 is the highest version supported by WebAssembly SIMD.
    env_jolt.Append(CCFLAGS=["-msse4.2"])
```

Reply:
> By the way, how does ARM support work if we're specifying SSE 4.2?

I think it's an emscripten compatibility layer thing. It accepts SSE 4.2 and setups the build to make it work, but internally, WebAssembly don't actually encode that information. I think it's up to the WebAssembly engines to enable (or not) SSE 4.2 if needed, but the wasm file don't need (or doesn't seem to, at least) that API per se.

Reply:
According to [this section](https://emscripten.org/docs/porting/simd.html#compiling-simd-code-targeting-x86-sse-instruction-sets) you may also need to pass `-msimd128` 🤷‍♂️